src/types/IJitsiMeetingOptions.ts

```diff
@@ -0,0 +1,83 @@
+export default interface IJitsiMeetingOptions {
```

Comment:
it would seem that most of these are not being used. Are they simply for documentation purposes?

Reply:
Yep 😃